### DISCUSSION TITLE
Resources for Learning FEA in Solid Mechanics

### DISCUSSION THREAD

**Discussion by sd-brady** [idaholab/moose#25725] [Q&A Modules: Solid mechanics]
Maybe this isn't the right location for this discussion. If it is not, let me know and I can take it down (or you're welcome to delete it).

I'd really like to learn more about the nuts and bolts of the mathematical theory behind solid mechanics FEA simulations. I've really been trying hard to read about FEA and plasticity, but it doesn't seem to be "clicking" for me yet. I have a good intuition for what is going on in the background but I don't have a great understanding for the actual system of equations and solution sequence that are being solved.

I have a copy of the "Introduction to Computational Plasticity" book that is often referenced here. It does seem like a good book that I'm able to mostly follow, but I just wish there were some more examples and problems to work through. I also have a hard time following complex mathematical proofs without good examples and problems to work through to put it together.

**To the seasoned moose solid mechanics experts: Is there a certain reference/references, detailed example problem, or university class/classes that helped bridge the gap for you**?

I'm just rambling at this point, but I think a good book idea would be describing in extreme detail one "cycle" of a simple 3D solid mechanics FEA problem. It could cover in detail something like a simple 3x3x3 3D mesh with some some simple boundary conditions. It could be broken up similar to how moose presents the Plug-n-Play system (i.e., 1) Compute Strain, 2) Compute the Elasticity Tensor, 3) Compute Stress... etc). The theory could be taught, but also, we could be working towards solving the one simple example problem throughout the book. 

TLDR - I want to understand but don't currently. I'd like to know the best references that helped you see the light.
